src/test/java/nextstep/courses/domain/SessionCoverImageTest.java

```diff
@@ -15,12 +14,15 @@
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
 
-class SessionCoverImageTest {
+public class SessionCoverImageTest {
```

Comment:
```suggestion
class SessionCoverImageTest {
```

public을 추가한 이유가 있나요?

junit5부터는 default 접근제한자로도 충분합니다.

[참고](https://stackoverflow.com/questions/55215949/why-junit-5-default-access-modifier-changed-to-package-private)

Reply:
`SessionCoverImageTest.SAMPLE_COVER_IMAGE` 픽스쳐를 다른 테스트에서 활용하기 위해 public을 붙여줬습니다!

Reply:
그런 이유라면 `*Fixture` 형태로 사용하시는게 조금 더 일반적입니다.
`SessionCoverImageFixture` 가 되겠네요.

이런 [플러그인](https://velog.io/@seok2/java-test-fixtures-%ED%94%8C%EB%9F%AC%EA%B7%B8%EC%9D%B8%EC%9D%84-%ED%99%9C%EC%9A%A9%ED%95%9C-%ED%85%8C%EC%8A%A4%ED%8A%B8-Fixtures-%EA%B4%80%EB%A6%AC)도 있으니 현업에서도 고려해보시면 좋은 요소입니다.